cmd/keymasterd/main.go

```diff
@@ -1706,10 +1710,11 @@ func (state *RuntimeState) u2fSignResponse(w http.ResponseWriter, r *http.Reques
 
 			_, err = state.updateAuthCookieAuthlevel(w, r, currentAuthLevel|AuthTypeU2F)
 			if err != nil {
-				logger.Printf("Autch Cookie NOT found ? %s", err)
+				logger.Printf("Auth Cookie NOT found ? %s", err)
 				state.writeFailureResponse(w, r, http.StatusInternalServerError, "Failure updating vip token")
 				return
 			}
+			eventNotifier.PublishAuthEvent(eventmon.AuthTypeU2F, authUser)
```

Comment:
For consistency on VIPAuthHandler and u2fSignResponse publish the auth event before or after updating the cookie.  (right now one is before and the other after)

Reply:
OK.